core/src/main/scala/scalaz/Heap.scala

```diff
@@ -409,7 +409,7 @@ object Heap extends HeapInstances {
         ins(f, link(f)(t, tp))(ts)
     }
 
-    def uniqify[A](f: (A, A) => Boolean): Forest[A] => Forest[A] = {
+    def uniquify[A](f: (A, A) => Boolean): Forest[A] => Forest[A] = {
```

Comment:
This is a public API. We should keep the original method for backward compatibility.

Reply:
I've added a deprecated method with the original name that calls this method: https://github.com/scalaz/scalaz/compare/3b0bbbf7de95f0b02288c57aa839462f2deca9e3..561f47ff91cdcc603cf9ac048b831e88bf504730

Does that do what you're looking for?

Are there other methods that need this treatment?